packages/twenty-front/src/modules/favorites/utils/calculateNewPosition.ts

```diff
@@ -0,0 +1,33 @@
+type CalculateNewPositionParams = {
+  destinationIndex: number;
+  sourceIndex: number;
+  items: Array<{ position: number }>;
+};
+
+export const calculateNewPosition = ({
```

Comment:
this looks duplicated code from other places. (table re-order, option menu fields re-order) I fill this could leave in the object-record/position module